conftest.py

```diff
@@ -2,7 +2,7 @@
 
 pytest_plugins = [
     # Plugins
-    "pytest_plugins.uncollector",
+    "pytest_plugins.rerun_rp.rerun_rp",
```

Comment:
instead of `rerun_rp` can't we can keep it as `rp` or `report_portal`. making folder as rerun_rp we are limiting it single activity. We could have further functions from the report portal that have different functionality. Just a suggestion, non-blocking from my side

Reply:
Basically, this plugin will only be hooked for rerunning the tests using RP data. If we will have something new to add for RP, it will be added in RP module and not in this plugin.

Even we have something else do to with RP results as pytest plugin, we will have a separate plugin for that. May be something like `block_rp` etc.